bitcoin_forks_test.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
```

Comment:
no Zcash tests in `bitcoin_forks_test`

Reply:
Oops, they were lost, added back.  Good catch.